libs/shared/sentry/src/lib/config-builder.ts

```diff
@@ -0,0 +1,87 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { Logger } from './sentry.types';
```

Comment:
Not sure the exact count. There's some of inversion of control going on in this lib so that sentry isn't directly depending on any one logger implementation. Probably another thing to clean up when we tackle logging, but for now I'm hoping this is okay.